README.md

```diff
@@ -3,7 +3,7 @@
 Tiny and framework-agnostic role-based permission management using
 a model of *composition* over inheritance.
 
-> `npm install --save permissionary`
+<blockquote>`npm install --save permissionary`</blockquote>
```

Comment:
@davidchambers I had to use `<blockquote>` tags here because the doctests try to evaluate markdown block quote syntax (`> My quote` will say `error: line 1: Unexpected identifier`).

Reply:
Could you do this?

    ```console
    $ npm install --save permissionary
    ```